client/lib/application/src/version-detail/version-detail.js

```diff
@@ -0,0 +1,48 @@
+import BaseView from 'common/src/base-view';
+import Template from './version-detail.hbs';
+import Flux from 'application/src/flux';
+import FetchResult from 'common/src/fetch-result';
+import ErrorTpl from 'common/src/error.hbs';
+import Placeholder from './placeholder.hbs';
+import Markdown from 'common/src/markdown';
+
+import 'common/asset/scss/application/application-detail.scss';
+
+class VersionDetail extends BaseView {
+    constructor( props ) {
+        this.stores = {
+            application: Flux.getStore('application'),
+        };
+        this.className = 'applicationDetail';
+        super(props);
+    }
+
+    update() {
+        let {applicationId} = this.props,
+            {versionId} = this.props;
+        this.data = {
+            ver: this.stores.application.getApplicationVersion( applicationId, versionId )
```

Comment:
This is a bit nitpicky, but would you mind changing this variable to `version`? It just reads better in the template.